ci/run_slither.sh

```diff
@@ -11,7 +11,7 @@ run_slither() {
     truffle compile 
 
     cd $PROTOCOL_DIR
-    slither --exclude=naming-convention,solc-version,pragma,external-function,reentrancy-benign,reentrancy-no-eth,arbitrary-send,locked-ether,reentrancy-eth,uninitialized-state-variables,incorrect-equality,reentrancy-events,assembly $1
+    slither --exclude=naming-convention,solc-version,pragma,external-function,reentrancy-benign,reentrancy-no-eth,arbitrary-send,locked-ether,reentrancy-eth,uninitialized-state-variables,incorrect-equality,reentrancy-events,assembly,shadowing-local $1
```

Comment:
I've been meaning to try Slither's triage mode [1] at some point, but haven't gotten around to it yet.

Definitely not part of this PR, but maybe we should add a task for triage Slither to the backlog?

1. https://github.com/crytic/slither/wiki/Usage#triage-mode